### PR TITLE
MBL-10397 Activity Start Date was getting decremented when toggling the ...

### PR DESCRIPTION
**Pull request by CHenrie**
...timeless option

### CODE REVIEW

**Comment by CHenrie** [comment]
@jbest84  Please Review

File: src/Views/Activity/Edit.js [hunk 1]
```diff
@@ -870,6 +866,26 @@ define('Mobile/SalesLogix/Views/Activity/Edit', [
...
+        _getNewStartDate: function (orginalStartDate, timeless){
+            var startDate,
+                currentTime;
+
+                if(!orginalStartDate){
+                    return null;
+                }
+                startDate = orginalStartDate;
+                if (timeless) {
+
```
Comment (jbest84) [inline src/Views/Activity/Edit.js]: Unnecessary newline

File: src/Views/Activity/Edit.js [hunk 2]
```diff
@@ -870,6 +866,26 @@ define('Mobile/SalesLogix/Views/Activity/Edit', [
...
+                    startDate = new Date(orginalStartDate.getUTCFullYear(), orginalStartDate.getUTCMonth(), orginalStartDate.getUTCDate(), 0, 0, 5);
+
```
Comment (jbest84) [inline src/Views/Activity/Edit.js]: Unnecessary newline

File: src/Views/Activity/Edit.js [hunk 3]
```diff
@@ -870,6 +866,26 @@ define('Mobile/SalesLogix/Views/Activity/Edit', [
...
+                } else {
+
+                    currentTime = moment();
+                    startDate = new Date(orginalStartDate.getUTCFullYear(), orginalStartDate.getUTCMonth(), orginalStartDate.getUTCDate(), currentTime.hours(), currentTime.minutes(), 0);
+
```
Comment (jbest84) [inline src/Views/Activity/Edit.js]: Unnecessary newline

File: src/Views/Activity/Edit.js [hunk 4]
```diff
@@ -870,6 +866,26 @@ define('Mobile/SalesLogix/Views/Activity/Edit', [
...
+                if(!orginalStartDate){
+                    return null;
+                }
+                startDate = orginalStartDate;
+                if (timeless) {
+
+                    startDate = new Date(orginalStartDate.getUTCFullYear(), orginalStartDate.getUTCMonth(), orginalStartDate.getUTCDate(), 0, 0, 5);
+
+                } else {
+
```
Comment (jbest84) [inline src/Views/Activity/Edit.js]: Unnecessary newline

File: src/Views/Activity/Edit.js [hunk 5]
```diff
@@ -870,6 +866,26 @@ define('Mobile/SalesLogix/Views/Activity/Edit', [
         formatDependentQuery: function(dependentValue, format, property) {
             return string.substitute(format, [utility.getValue(dependentValue, property || '$key')]);
         },
+        _getNewStartDate: function (orginalStartDate, timeless){
+            var startDate,
+                currentTime;
+
+                if(!orginalStartDate){
```
Comment (jbest84) [inline src/Views/Activity/Edit.js]: Put space after if, and before the {

`if (something) {`

**Pull request merged**